mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/loanchargedialog/LoanChargeDialogFragment.java

```diff
@@ -247,7 +247,7 @@ public void onNothingSelected(AdapterView<?> parent) {
 
     @Override
     public void showLoanChargesCreatedSuccessfully(Charges charges) {
-        Toast.makeText(getActivity(), "Charge created successfully", Toast
+        Toast.makeText(getActivity(), R.string.charge_created, Toast
```

Comment:
Still, there are two string resources with same value 
1. ```<string name="charge_created_success">Charge created successfully</string>```
2. ```<string name="charge_created">Charge created successfully</string>```

Stay with 1st string and use this.